src/services/jwt.service.js

```diff
@@ -0,0 +1,39 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+
+function sign(user) {
+  const { id, email } = user;
+  const token = jwt.sign({ id, email }, process.env.JWT_KEY);
+
+  return token;
+}
+
+function verify(token) {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (e) {
+    return null;
+  }
+}
+
+function signRefresh(user) {
+  const { id, email } = user;
+  const token = jwt.sign({ id, email }, process.env.JWT_REFRESH_KEY);
```

Comment:
Similar to the access token, consider adding an expiration time to the refresh token to enhance security.